flutter_local_notifications/lib/src/flutter_local_notifications_plugin.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:async';
 import 'dart:io';
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter_local_notifications/src/platform_specifics/android/notification_channel.dart';
```

Comment:
Relative imports are being used within the plugin and should be placed after package imports. See https://dart.dev/guides/language/effective-dart/usage#prefer-relative-paths-when-importing-libraries-within-your-own-packages-lib-directory